### ISSUE TITLE
Printing of Betti Tables

### ISSUE THREAD

**Issue by wdecker** [oscar-system/Oscar.jl#2793]
The printing of Betti tables is somewhat shifted if a degree in the first column is negative
(I already use functionality provided by PR #2500).

`R, (x, y) = graded_polynomial_ring(QQ, ["x", "y"]);`
`I = ideal(R, [x, y, x+y]);`
`M = quotient_ring_as_module(I)`
`FM = free_resolution(M, algorithm = :nres)`
`betti_table(FM)`

**Comment by fingolfin** [comment]
For reference, this is what is printed right now:
```
julia> R, (x, y) = graded_polynomial_ring(QQ, ["x", "y"]);

julia> I = ideal(R, [x, y, x+y]);

julia> M = quotient_ring_as_module(I)
Graded subquotient of submodule of R^1 generated by
1 -> e[1]
by submodule of R^1 generated by
1 -> x*e[1]
2 -> y*e[1]
3 -> (x + y)*e[1]

julia> FM = free_resolution(M, algorithm = :nres)
Free resolution of M
R^1 <---- R^3 <---- R^2 <---- 0
0         1         2         3

julia> betti_table(FM)
       0 1 2
------------
-1    : - - 1
0    : 1 3 1
------------
total: 1 3 2
```

There are also some alignment issues due to the minus sign.

It would easy to solve this using the PrettyTables package (but I think @fieker doesn't want us to use it?)

Note that character tables also have to deal with similar issues (and are doing so successfully, it seems), so it might be useful to look at the `function Base.show(io::IO, ::MIME"text/plain", tbl::GAPGroupCharacterTable)` method by @ThomasBreuer  ?

**Comment by fingolfin** [comment]
@ederc just said that maybe he and @RafaelDavidMohr can look into it (if time permits)

**Comment by RafaelDavidMohr** [comment]
Ok, I will have a look!

**Comment by wdecker** [comment]
Please also check what happens if we have integers with more than one digit.

**Comment by ThomasBreuer** [comment]
The function used by the `show` function for character tables is `labelled_matrix_formatted`. Perhaps it would be "overkill" to use this function for Betti tables, but maybe not.

**Comment by RafaelDavidMohr** [comment]
Thanks for the suggestions, me and @ederc discussed that the easiest would be probably be to use the recently merged code for sheaf cohomology tables for betti tables as well, they have almost the exact same format. If no one has any objections to this I will create a corresponding PR tomorrow.

**Comment by thofma** [comment]
(Please no dependency on PrettyTables.)

**Comment by jankoboehm** [comment]
Note that the printing code for Betti tables also can print in reverse horizontal direction and has already code for multigradings and projections thereof. So I would suggest that we rather do the very minor fix than to replace it.

**Comment by afkafkafk13** [comment]
fixed by PR #2834
